its/src/test/java/com/sonar/it/csharp/DoNotAnalyzeTestFilesTest.java

```diff
@@ -36,35 +36,54 @@
   @Rule
   public TemporaryFolder temp = TestUtils.createTempFolder();
 
-  private static final String PROJECT = "DoNotAnalyzeTestFilesTest";
+  private static final String CSHARP_ONLY_TEST_PROJECT = "DoNotAnalyzeTestFilesTest";
+  // the below is explicitly marked as test with <SonarQubeTestProject> MSBuild property
+  private static final String EXPLICITLY_MARKED_AS_TEST = "HtmlCSharpExplicitlyMarkedAsTest";
 
   @Before
   public void init() {
     TestUtils.reset(ORCHESTRATOR);
   }
 
   @Test
-  public void should_not_increment_test() throws Exception {
-    BuildResult buildResult = Tests.analyzeProjectWithSubProject(temp, PROJECT, "MyLib.Tests", "no_rule");
+  public void with_csharp_only_test_should_not_increment_test() throws Exception {
```

Comment:
What about `with_csharp_only_test_should_not_populate_metrics`? increment test seems confusing for me.